src/styles/theme/fonts.tsx

```diff
@@ -0,0 +1,27 @@
+export const dm_sans = "DM Sans, sans-serif";
+export const dm_serif_display = "DM Serif Display, serif";
+
+export const GoogleFontImport = () => (
```

Comment:
🎉 love this. having this is awesome. I think one of the future steps could be documenting the how to customize this somewhere.